packages/wallet/src/TxBuilder/buildTx.ts

```diff
@@ -91,13 +97,18 @@ export class ObservableWalletTxBuilder implements TxBuilder {
   #outputValidator: OutputValidator;
   #delegateConfig: DelegateConfig;
   #logger: Logger;
+  #enabled = true; // Do not allow building if a transaction built by this builder was already submitted
 
   constructor({ observableWallet, outputValidator = createWalletUtil(observableWallet), logger }: BuildTxProps) {
     this.#observableWallet = observableWallet;
     this.#outputValidator = outputValidator;
     this.#logger = logger;
   }
 
+  isDestroyed(): boolean {
```

Comment:
Let's keep the terminology consistent, I like this one (from the error name) better 
```suggestion
  isSubmitted(): boolean {
```

Reply:
Updated here: [e2a9138](https://github.com/input-output-hk/cardano-js-sdk/pull/434/commits/e2a9138bb61064f99b1d37cfbabfdffa81a77fac)